hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/ITestAbfsClient.java

```diff
@@ -579,8 +577,6 @@ public void testExpectHundredContinue() throws Exception {
         .isEqualTo(0);
 
     // Verify that the same request was retried with expect header disabled.
-    Assertions.assertThat(appendRequestParameters.isExpectHeaderEnabled())
-        .describedAs("The expect header is not false")
-        .isEqualTo(false);
+    assertFalse(appendRequestParameters.isExpectHeaderEnabled());
```

Comment:
sorry, i must have explained it wrong. AssertJ has assert/true false too in its type `AbstractBooleanAssert`. you can go
```
Assertions.assertThat(appendRequestParameters.isExpectHeaderEnabled())
  .describedAs("The expect header is not false")
  .isFalse()
```
so can you restore what was cut and just change the assert.
```